packages/preview/modern-ysu-thesis/0.1.0/lib.typ

```diff
@@ -0,0 +1,308 @@
+// 南京大学学位论文模板 modern-nju-thesis
+// Author: https://github.com/OrangeX4
+// Repo: https://github.com/nju-lug/modern-nju-thesis
+// 在线模板可能不会更新得很及时，如果需要最新版本，请关注 Repo
+
+#import "layouts/doc.typ": doc
+#import "layouts/preface.typ": preface
+#import "layouts/mainmatter.typ": mainmatter
+#import "layouts/appendix.typ": appendix
+#import "pages/fonts-display-page.typ": fonts-display-page
+#import "pages/bachelor-cover.typ": bachelor-cover
+#import "pages/bachelor-detail.typ": bachelor-detail
+#import "pages/master-cover.typ": master-cover
+#import "pages/bachelor-decl-page.typ": bachelor-decl-page
+#import "pages/master-decl-page.typ": master-decl-page
+#import "pages/bachelor-abstract.typ": bachelor-abstract
+#import "pages/master-abstract.typ": master-abstract
+#import "pages/bachelor-abstract-en.typ": bachelor-abstract-en
+#import "pages/master-abstract-en.typ": master-abstract-en
+#import "pages/bachelor-outline-page.typ": bachelor-outline-page
+#import "pages/list-of-figures.typ": list-of-figures
+#import "pages/list-of-tables.typ": list-of-tables
+#import "pages/notation.typ": notation
+#import "pages/acknowledgement.typ": acknowledgement
+#import "utils/custom-cuti.typ": *
+#import "utils/bilingual-bibliography.typ": bilingual-bibliography
+#import "utils/custom-numbering.typ": custom-numbering
+#import "utils/custom-heading.typ": heading-display, active-heading, current-heading
+#import "utils/indent.typ": indent
+#import "@preview/i-figured:0.2.4": show-figure, show-equation
+#import "utils/style.typ": 字体
+#import "utils/style.typ": 字号
+
+#import "layouts/page-header.typ": page-header
+
```

Comment:
Removing the logo works too, but you could also add it here and have the package work as you intended:

```suggestion
#let logo = image.with("assets/vi/ysulogo.png")

```